### ISSUE TITLE
XPCOMGlueLoad error for file /usr/lib64/firefox/libxul.so: libssl3.so: version `NSS_3.27' not found

### ISSUE THREAD

**Issue by wcerfgba** [standardnotes/desktop#79]
When trying to launch the Dashboard in the desktop app (e.g. through Extensions > Open Dashboard), nothing happens, and I get the following in my terminal:
```
installed: X-AppImage-BuildId=e4e17250-55d3-11a7-22e5-4bde9a38da74 image: X-AppImage-BuildId=e4e17250-55d3-11a7-22e5-4bde9a38da74
XPCOMGlueLoad error for file /usr/lib64/firefox/libxul.so:
/tmp/.mount_mFxzSO/usr/lib/libssl3.so: version `NSS_3.27' not found (required by /usr/lib64/firefox/libxul.so)
Couldn't load XPCOM.
```

I am not familiar with how AppImage works (seems cool though, thanks for introducing me to it!), but I assume I can't install the appropriate library on my system because it is running in its own file system (`/tmp/.mount_mFxzSO`)?

Thanks for your help, I'm really happy with Standard Notes!

**Comment by mobitar** [comment]
Hey John, glad to hear you're happy with it! That's a strange issue. Do the other external links work, or is it just this one that causes problems?

**Comment by wcerfgba** [comment]
@mobitar Happens with the Help link also. I am running Fedora rawhide kernel `4.12.0-0.rc7.git2.1.fc27.x86_64` with `nss-3.31.0-2.fc27`.

**Comment by mobitar** [comment]
From the looks of it, it seems that the app is trying to open Firefox but failing to do so:

https://www.google.com/search?q=Couldn%27t+load+XPCOM.&oq=Couldn%27t+load+XPCOM.&aqs=chrome..69i57.141j0j7&sourceid=chrome&ie=UTF-8

Can you try some of those suggestions and see if it works?

**Comment by wcerfgba** [comment]
Hi @mobitar , sorry for the delay.

I had a look at the XPCOM suggestions, and looked for similar errors related XPCOMGlueLoad, libxul, and libssl/libnss, but most of what I found were old threads and bug reports related to library package version incompatibilities. 

This is fixed for me with [the v1.2.0 release](https://github.com/standardnotes/desktop/releases/tag/v1.2.0), so I'll close this now.
